pure-eth-connector/Makefile

```diff
@@ -1,11 +1,34 @@
+NEAR_ACC=nostd01.eth-connector.testnet
+#NEAR_ACC=dev-1616437086828-5300267
+
 build:
 	@./build.sh
+	@cp ~/dev/rs/near/eth-connector/pure-eth-connector/Makefile ~/dev/vagrant/near/ || true
+
+clippy:
+	@cargo clippy
 
 check:
-	@cargo build
+	@cargo check
```

Comment:
include `--all-features` arg to ensure it covers everything. You could also add `--all-targets` if you want but I do not think we have multiple different targets right now.